Composer/packages/server/src/models/bot/luPublisher.ts

```diff
@@ -208,7 +208,7 @@ export class LuPublisher {
 
     //add the lu file that are not in interuption folder.
     files.forEach((file) => {
-      if (!~paths.indexOf(file.name)) {
+      if (!paths.includes(file.name)) {
```

Comment:
This becomes a O(n^2) with the lookup inside for loop. Can paths be a set?

Reply:
Figured most of this code was already there. If u have time for a little cleanup will be great

Reply:
Will do.